docs/System-Description-Format.md

```diff
@@ -605,21 +666,31 @@ JSON Example:
 
 JSON Example:
 ```json
-  "repositories": [
-    {
-      "alias": "SLE-12-SLP",
-      "name": "SLE-12-SLP",
-      "type": "yast2",
-      "url": "http://dist.suse.de/install/SLP/SLE-12-Server-Beta10/x86_64/DVD1/",
-      "enabled": true,
-      "autorefresh": false,
-      "gpgcheck": true,
-      "priority": 99,
-      "package_manager": "zypp"
-    }
-  ]
+  "repositories": {
+    "_attributes": {
+      "repository_system": "zypp"
+    },
+    "_elements": [
+      {
+        "alias": "download.opensuse.org-oss",
+        "name": "Main Repository (OSS)",
+        "type": "yast2",
+        "url": "http://download.opensuse.org/distribution/leap/42.1/repo/oss/",
+        "enabled": true,
+        "autorefresh": true,
+        "gpgcheck": true,
+        "priority": 99
+      }
+    ]
+  }
 ```
 
+The attributes of the repositories array are:
+
+| item            | description                          | type                       |
+|-----------------|--------------------------------------|----------------------------|
+| repository_system | The repository manager (e.g. "zypp" or "apt") | string           |
```

Comment:
This is an enum, not a custom string, right? There is an exact list of allowed values ("zypp" and "apt").